Makefile

```diff
@@ -106,6 +106,14 @@ set-kcm-version: yq
 set-kcm-repo: yq
 	$(YQ) eval '.image.repository = "$(IMG_REPO)"' -i $(PROVIDER_TEMPLATES_DIR)/kcm/values.yaml
 
+.PHONY: set-telemetry-repo
+set-telemetry-repo: yq
+	$(YQ) eval '.regional.telemetry.controller.image.repository = "$(IMG_TELEMETRY_REPO)"' -i $(PROVIDER_TEMPLATES_DIR)/kcm/values.yaml
```

Comment:
It's better to set this in the `kcm-regional` chart directly. Versions of kcm and kcm-regional are now synced + this will potentially have an impact when regional deployed separately.